### ISSUE TITLE
[Bug]: Avatar sprite is changed inside dungeons

### ISSUE THREAD

**Issue by milotic** [pokeclicker/pokeclicker#5428]
### Confirmations

- [X] I have searched open and closed issues for any existing report of my issue

### What happened?

the avatar sprite in the world is changed to other sprite inside dungeons

female avatar in the world:

![Captura de pantalla 2024-07-11 233120](https://github.com/user-attachments/assets/b3dded39-67fd-4346-834a-b7d2e3aa8f1e)

battle frontier inside dungeons:

![Captura de pantalla 2024-07-11 233146](https://github.com/user-attachments/assets/bf68c9ef-7e85-4aba-a048-aae9525a1e81)

### What did you expect to happen?

your avatar persist in the gameplay

### Steps to Reproduce

1. open a dungeon
2. your avatar is changed inside

### What operating system are you using?

windows 11

### What browsers are you seeing the problem on?

Desktop Client

**Comment by milotic** [comment]
save with the bug:

[[v0.10.19] PokeClicker 2024-06-18 21_20_52 Backup.txt](https://github.com/user-attachments/files/16189112/v0.10.19.PokeClicker.2024-06-18.21_20_52.Backup.txt)

**Comment by RegisCoaxans** [comment]
Does it happen consistently ?
I launched your file and it worked fine on every dungeon try for me.
Play on the latest version (0.10.21 compared to 0.10.20) maybe.

**Comment by milotic** [comment]
Yes, it happened always since the last update until I took a new avatar and it ignores the file name, I always overwrite the save file.
Reseting the game dont fix the problem and the only way to fix the avatar was taking a new avatar.

if loading the save file is ok, maybe is a sort of cache in the client? because i play everyday but do the save like every 3 days?

**Comment by milotic** [comment]
@RegisCoaxans the bug happen in a new place not related with dungeons, updated the issue with the new case
